region/maridia/inner-pink.json

```diff
@@ -1440,6 +1440,78 @@
+                  "note": [
+                    "Stand on the appropriate pixel for shooting diagonally through the wall and use the microwave trick to defeat Botwoon.",
+                    "Using angle up, it is where Samus' front foot is on the seam in the floor.",
+                    "There is not proper spacing for landing an angle down shot and xraying.",
+                    "Waiting for Botwoon to peak their head through the wall works too but is less safe."
+                  ]
+                },
+                {
+                  "name": "Back-Side Botwoon Magic Pixel",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateUnderwater",
+                    {
+                      "enemyKill": {
+                        "enemies": [ [ "Reverse Botwoon" ] ],
+                        "explicitWeapons": [ 
+                          "Charge+Plasma", 
+                          "Charge+Ice+Spazer"
+                       ]
+                      }
+                    }
+                  ],
+                  "reusableRoomwideNotable": "Back-Side Botwoon Magic Pixel Beam Fight",
+                  "note": [
+                    "A charge beam shot will pass right through the dividing wall if fired from the correct 2pixel window.",
+                    "Using angle down the spot is where Samus' front toe touches the wall.",
+                    "Using angle up, it is where Samus' front foot is on the seam in the floor."
+                  ]
+                },
+                {
+                  "name": "Back-Side Botwoon Plasma Shield Fight",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateUnderwater",
+                    {
+                      "enemyKill": {
+                        "enemies": [ [ "Reverse Botwoon" ] ],
+                        "explicitWeapons": [ "Plasma Shield" ]
+                      }
+                    }
+                  ],
+                  "note": "Stand near the dividing wall and unleash the Plasma Special Beam Attack!",
+                  "devNote": "6 power bombs expected."
+                },
+                {
+                  "name": "Back-Side Botwoon Super Only Fight",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateUnderwater",
+                    "canBePatient",
+                    {"enemyKill": {
+                      "enemies": [ [ "Reverse Botwoon" ] ],
+                      "explicitWeapons": [ "Super" ]
+                    }},
+                    {"ammo": {
+                      "type": "Super",
+                      "count": 3
+                    }}
+                  ],
+                  "note": "Wait for the one pattern (bottom->right) where Botwoon's head passes through the dividing barrier briefly and fire a Super Missile.  This takes a long time.",
```

Comment:
This would be a good place to mention that it's random, but averages to approximately 1 super per minute.